### DISCUSSION TITLE
Lightning effect?

### DISCUSSION THREAD

**Discussion by DannyWagensveld** [theyosh/TerrariumPI#919] [Q&A] [answered]
Hello,

Just starting with the build of a paludarium, where i will use TerrariumPI. Everything looks very good so far but I was thinking; is there a possibility to create a  thunder/lightning effect? Think this would be fun in combination with the sound effects.

**Comment by theyosh** [comment, marked as answer]
Hmm, I think that will be pretty difficult to support directly. But indirectly...

https://www.youtube.com/watch?v=NEgEiESV_dk
https://duckduckgo.com/?t=lm&q=Velleman+MK186

or complete systems: 
https://www.frightprops.com/lighting/thunder-lightning-controllers/budget-lightning-controllers/perfect-storm.html
https://www.amazon.com/Lighting-Control-Operation-Portable-Christmas/dp/B06XQ88WC4

**Reply by DannyWagensveld** [reply]
Thanks, I will have a look!
